helper.js

```diff
@@ -178,9 +178,50 @@ function createIframe(opt, width, height) {
   return el
 }
 
+function wifiChannelAlias(ch) {
```

Comment:
In infobox/node

On June 6, 2016 3:01:14 PM GMT+02:00, eberhab <notifications@github.com> wrote:
>> @@ -178,9 +178,50 @@ function createIframe(opt, width, height) {
>>    return el
>>  }
>>  
>> +function wifiChannelAlias(ch) {
>
>I guess not. It felt like a good place for it. Where would you like to
>have it?
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/plumpudding/hopglass/pull/38/files/8618656b1bb4d339648c1838629af9aada6ef4c5#r65885189

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.